src/packer.pkr.hcl

```diff
@@ -132,12 +132,6 @@ source "amazon-ebs" "openvpn" {
 build {
   sources = ["source.amazon-ebs.openvpn"]
 
-  provisioner "ansible" {
-    playbook_file = "src/upgrade.yml"
-    use_proxy     = false
-    use_sftp      = true
-  }
-
```

Comment:
I don't think this should be removed.  We want the upgrade to happen _before_ the `src/python.yml` playbook runs.